app/views/fincap_templates/index.html.erb

```diff
@@ -4,31 +4,25 @@
       img_src: template.hero_image_component, 
       description: template.hero_description_component 
     %>
-    <div class='row'>
-      <div class="body-content">
-        <%= template.body.html_safe %>
-      </div>
+    <div class="body-content">
+      <%= template.body.html_safe %>
     </div>
   <% end %>
 
   <% present_collection(resource_collection).each do |template| %>
-    <div class='row'>
-      <div class='l-constrained-padded'>
-        <div class='teaser-box--thematic-review'>
-          <img class='teaser-box__image'
-                 src=<%= template.hero_image_component %>
-                 alt=''
-            />
-          <div class='teaser-box__content'>
-            <h3 class='teaser-box__title'>Thematic Review:</h3>
-            <a href= "<%= template.full_path %>" class='teaser-box__content-link'>
-              <%= template.title %>
-            </a>
-            <p class='teaser-box__content-text'>
-              <%= template.overview_content.html_safe %>
-            </p>
-          </div>
-        </div>
+    <div class='teaser-box--thematic-review'>
+      <img class='teaser-box__image'
+             src=<%= template.hero_image_component %>
+             alt=''
+        />
+      <div class='teaser-box__content'>
+        <h3 class='teaser-box__title'>Thematic Review:</h3>
+        <a href= "<%= template.full_path %>" class='teaser-box__content-link'>
+          <%= template.title %>
+        </a>
```

Comment:
Can you change this line to: `<%= link_to template.title, template.full_path, class: "teaser-box__content-link"  %>`